Source/Core/Core/HW/EXI/BBA/BuiltIn.h

```diff
@@ -0,0 +1,110 @@
+#pragma once
```

Comment:
This file is missing the license comment currently:

```C++
// Copyright 2022 Dolphin Emulator Project
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
added!